_posts/2020-12-04-status-update.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "2020-12-04: IAC Team Status Update"
+author: daianamezdrea
+categories:
+  - team
+  - status
+tags:
+-community
+-ci cloud
+-module releases
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-kubernetes#451`][puppetlabs-kubernetes-pr-451]: "Add scheduler_extra_arguments parameter", thanks to [treydock][treydock]
+- [`action-litmus_parallel#71`][action-litmus_parallel-pr-71]: "(MAINT) Fix user install bug", thanks to [RandomNoun7][RandomNoun7]
+- [`action-litmus_parallel#69`][action-litmus_parallel-pr-69]: "Fix ENV Set Method", thanks to [RandomNoun7][RandomNoun7]
+- [`action-litmus_spec#61`][action-litmus_spec-pr-61]: "(MAINT) Fix user install bug", thanks to [RandomNoun7][RandomNoun7]
+- [`action-litmus_spec#60`][action-litmus_spec-pr-60]: "Fix ENV Set Method", thanks to [RandomNoun7][RandomNoun7]
+- [`pdk-templates#360`][pdk-templates-pr-360]: "(fix) Set rubocop ruby version to 2.4", thanks to [tuxmea][tuxmea]
+- [`puppet-modulebuilder#26`][puppet-modulebuilder-pr-26]: "Use match_path instead of match_paths", thanks to [ekohl][ekohl]
+- [`puppet_litmus#344`][puppet_litmus-pr-344]: "Fix append_cli parameters", thanks to [hajee][hajee]
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-concat`][puppetlabs-concat] (`6.3.0`)
+- [`puppetlabs-acl`][puppetlabs-acl] (`3.2.1`)
+- [`puppetlabs-kubernetes`][puppetlabs-kubernetes] (`5.4.0`)
+- [`puppetlabs-device_manager`][puppetlabs-device_manager] (`3.1.0`)
+
+## Triage day
+
+Even if half of the team was on PTO, we got through all the supported modules repos and most of all our tooling processing, so we managed to review and merge 11 PRs and we released 4 modules. 
+
+## CI and special Wednesday
+
+We aplied again the principle "Team work, dream work" and together we ported successfully 22 modules onto the CI. Many thanks to the entire team for the hard work! Check out the new look of our modules: https://github.com/puppetlabs/puppetlabs-chocolatey/pull/235/checks
+
+## Happy 3 months for Disha! 
+
+It's been 3 months since Disha joined the team and she started her journey in our team. If you want to know more about her work and last updates, you can always check [here](https://puppetlabs.github.io/iac/lifeofinternatpuppet/post_14.html) her blog.
```

Comment:
```suggestion
It's been 3 months since Disha joined the team and she started her journey in our team. If you want to know more about her work and last updates, you can always check her blog [here](https://puppetlabs.github.io/iac/lifeofinternatpuppet/post_14.html).
```